<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZDJLDialog</class>
 <widget class="QDialog" name="ZDJLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>诊断记录管理</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>961</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>记录编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>故障代码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>故障名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>产品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_start">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>true</bool>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2015</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_end">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="currentSectionIndex">
    <number>2</number>
   </property>
   <property name="date">
    <date>
     <year>2015</year>
     <month>11</month>
     <day>11</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>560</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>630</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>781</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_del">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>490</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
